pkg/operator/vttablet/vtbackup_pod.go

```diff
@@ -179,6 +179,11 @@ func NewBackupPod(key client.ObjectKey, backupSpec *BackupSpec) *corev1.Pod {
 		pod.Spec.ServiceAccountName = planetscalev2.DefaultVitessServiceAccount
 	}
 
+	// Allow users to inject their own affinities.
+	if tabletSpec.Affinity != nil {
+		pod.Spec.Affinity = tabletSpec.Affinity
```

Comment:
Since there is no default Affinity for vtbackup, and this is a New function (not Update on an existing Pod), you can just put `Affinity: tabletSpec.Affinity,` in the PodSpec literal above, like we do for `Tolerations`.

Reply:
Good point, moving.

Reply:
Done.